.github/workflows/autoformat.yml

```diff
@@ -70,5 +72,6 @@ jobs:
       - run: vendor/bin/php-cs-fixer fix
 
       - uses: stefanzweifel/git-auto-commit-action@v5
+        if: github.ref == 'refs/heads/master'
```

Comment:
@spawnia would you be fine with running the actions always? Except running the fix on master only?

I'd like to ensure the checks always pass. Then I can tell Renovate to automerge fixer if CI is green.

Reply:
I would like the automatic fixes to happen in pull requests, and preferrably not on master. Keeps the commit history clean.